packages/components/fields/select-field/src/select-field.tsx

```diff
@@ -196,7 +196,7 @@ export type TSelectFieldProps = {
    * <br/>
    * The value will be the selected option, or an array of options in case isMulti is true.
    */
-  onChange?: (event?: TCustomEvent) => void;
+  onChange?: (event: TCustomEvent) => void;
```

Comment:
The event was marked as optional but this callback is just passed internally to the `SelectInput` component where the event is not optional.